### PR TITLE
[close #253] add input event metric for puller

### LINKED ISSUE #253: cdc: Add input event metric for puller
## Feature Request

**Is your feature request related to a problem? Please describe:**

TiKV-CDC: Add input event metric for puller

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?
add input event count metric for puller

Issue Number: close #253 

Problem Description: **TBD**

### What is changed and how does it work?
add input event count metric for puller

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

### CODE REVIEW

**Comment by zeminzhou** [comment]
image
image
image

**Comment by zeminzhou** [comment]
PTAL~, Thanks

File: cdc/metrics/grafana/tikv-cdc.json [hunk 1]
```diff
@@ -9661,7 +9948,10 @@
               "interval": "",
               "refId": "D",
               "hide": false,
-              "intervalFactor": 2
+              "interval": "",
```
Comment (pingyu) [inline cdc/metrics/grafana/tikv-cdc.json:9951]: It is duplicated.

File: cdc/metrics/grafana/tikv-cdc.json [hunk 2]
```diff
@@ -9661,7 +9948,8 @@
               "interval": "",
               "refId": "D",
               "hide": false,
-              "intervalFactor": 2
+              "intervalFactor": 2,
+              "refId": "D"
```
Comment (pingyu) [inline cdc/metrics/grafana/tikv-cdc.json:9952]: "refId" is duplicated. Rest LGTM.

File: cdc/metrics/grafana/tikv-cdc.json (hunk 1 above)
Comment (zeminzhou) [reply cdc/metrics/grafana/tikv-cdc.json]: removed, thanks!

File: cdc/metrics/grafana/tikv-cdc.json (hunk 2 above)
Comment (zeminzhou) [reply cdc/metrics/grafana/tikv-cdc.json]: removed, thanks, PTAL~

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by haojinming — APPROVED** [review]
LGTM

**Pull request merged**